.github/workflows/update_gh_pages.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow updates GitHub Pages on the repo it is run from.
+
+name: Update GitHub Pages
+
+on:
+  workflow_dispatch:  # Manually trigger. Colon is required.
+
+permissions:
+  contents: write  # For checkout and push.
+
+jobs:
+  update-github-pages:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        ref: master
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: NPM install
+      # Use CI so that we don't update dependencies in this step.
+      # TODO: remove --force flag from installing examples when examples dependencies are routinely tested
```

Comment:
I see @BeksOmega suggested a bug for this.  If you open one, please mention this TODO (and make it `TODO(#xxxx):…`).